lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -237,6 +237,24 @@ TF_DEFINE_PRIVATE_TOKENS(
     (ND_swizzle_vector4_vector2)
     (ND_swizzle_vector4_vector3)
     (ND_swizzle_vector4_vector4)
+    // Conversion nodes:
+    (ND_convert_float_color3)
+    (ND_convert_float_color4)
+    (ND_convert_float_vector2)
+    (ND_convert_float_vector3)
+    (ND_convert_float_vector4)
+    (ND_convert_vector2_vector3)
+    (ND_convert_vector3_color3)
+    (ND_convert_vector3_vector2)
+    (ND_convert_vector3_vector4)
+    (ND_convert_vector4_color4)
+    (ND_convert_vector4_vector3)
+    (ND_convert_color3_vector3)
+    (ND_convert_color4_vector4)
+    (ND_convert_color3_color4)
+    (ND_convert_color4_color3)
+    (ND_convert_boolean_float)
+    (ND_convert_integer_float)
```

Comment:
These nodes also affect topology, as found while trying the brick procedural MaterialX demo file.

Reply:
where are these tokens use?

Reply:
They are used to recognize nodes that affect the topology of the shader graph when we compute the graph key for shader re-use. I used a MaterialX API to simplify the list in the latest commit.